gradle/libs.versions.toml

```diff
@@ -0,0 +1,231 @@
+[versions]
+agp = '8.1.0'
+androidInstallReferrer = '2.2'
+androidxActivity = '1.8.0'
+androidxAppcompat = '1.6.1'
+androidxArchCore = '2.2.0'
+androidxCamera = '1.3.4'
+androidxCardview = '1.0.0'
+androidxComposeMaterial3 = '1.1.1'
+androidxComposeNavigation = '2.7.6'
```

Comment:
Minor (🔍): Consider renaming that version to `androidxNavigationCompose` instead. This version + dependency is more `navigation` than `compose`.

PS: Same applies to its related dependency, the `androidx-navigation`, which should be renamed to `androidx-navigation-compose`.

Reply:
Fixed